owlery.py

```diff
@@ -53,9 +38,9 @@ def run_update(self, template):
             'password': self.password,
             'update': template
         }
-        data = urllib.urlencode(payload)
-        response = urllib.urlopen(self.update_endpoint, data)
-        return response.code
+        url = self.update_endpoint
+        response = requests.post(url, params=payload)
```

Comment:
Does this work as a post body? It seems weird to be a url-encoded thing that is being posted. Can you verify if it has to be params or if it can be in the post body?